flac.md

```diff
@@ -18,9 +18,15 @@ FLAC owes much to the many people who have advanced the audio compression field
 
 # Scope
 
-It is a known fact that no algorithm can losslessly compress all possible input, so most compressors restrict themselves to a useful domain and try to work as well as possible within that domain. FLAC's domain is audio data. Though it can losslessly code any input, only certain kinds of input will get smaller. FLAC exploits the fact that audio data typically has a high degree of sample-to-sample correlation.
+FLAC stands for Free Lossless Audio Compressor: it is designed to reduce the amount of computer storage space needed to store digital audio signals (i.e. compression), without needing to remove information in doing so (i.e. lossless). FLAC is free in the sense that its specification is open, its reference implementation is open-source and it is not encumbered by any known patent.
```

Comment:
Is FLAC not Free Lossless Audio **Codec**?

Reply:
Huh, I don't know how I ended up writing this down. Good catch.

```suggestion
FLAC stands for Free Lossless Audio Codec: it is designed to reduce the amount of computer storage space needed to store digital audio signals without needing to remove information in doing so (i.e. lossless). FLAC is free in the sense that its specification is open, its reference implementation is open-source and it is not encumbered by any known patent.
```